app/Jobs/DeleteWikiDbJob.php

```diff
@@ -107,6 +107,9 @@ public function handle( DatabaseManager $manager )
                 $replacedCount = 0;
                 $tableWithoutPrefix = str_replace($wikiDB->prefix . '_', '', $table, $replacedCount );
                 if ($replacedCount !== 1) {
+                    /**
+                     * @psalm-suppress InvalidCast
```

Comment:
I really tried hard to understand these warnings but I could not make any sense of them so I suppressed them for now. If anyone has a hunch why they might show up, please lmk.